src/pages/public_registration/LogPanel.tsx

```diff
@@ -41,10 +56,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     cacheTime: 1_800_000, // 30 minutes
   });
 
+  const logs: LogList = [...tickets];
+
+  if (registration.publishedDate) {
+    const registrationPublished: NonTicketLogEntry = {
+      modifiedDate: registration.publishedDate,
+      description: t('registration.status.PUBLISHED_METADATA'),
+    };
+    logs.push(registrationPublished);
+  }
+
+  if (registration.publishedDate && registration.publishedDate < registration.modifiedDate) {
```

Comment:
Trenger vi tenke på publiseringsdato her? jeg ville antatt man ønsker å vise modifiseringsdato uavhengig av publiseringstatus

```suggestion
  if (registration.modifiedDate) {
```

Reply:
PO ønsket spesifikt at denne kun skal vises dersom det er gjort endring etter at registreringen er publisert